dev-libs/protobuf/protobuf-3.0.0_beta3_p1.ebuild

```diff
@@ -0,0 +1,101 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 python3_3 python3_4 python3_5 )
+inherit python-r1 autotools flag-o-matic toolchain-funcs elisp-common multilib-minimal
+
+# If you bump this package, also consider bumping the official language bindings!
+# At the current time these are java and python.
+MY_PV=${PV/_beta/-beta-}
+MY_PV=${MY_PV/_p/.}
+
+DESCRIPTION="Google's Protocol Buffers -- an efficient method of encoding structured data"
+HOMEPAGE="https://github.com/google/protobuf/ https://developers.google.com/protocol-buffers/"
+SRC_URI="https://github.com/google/protobuf/archive/v${MY_PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0/10"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~mips ~ppc ~ppc64 ~sh ~sparc ~x86 ~amd64-linux ~arm-linux ~x86-linux ~x64-macos ~x86-macos"
+IUSE="emacs examples java python static-libs test vim-syntax zlib"
+
+DEPEND="emacs? ( virtual/emacs )
+	zlib? ( >=sys-libs/zlib-1.2.8-r1[${MULTILIB_USEDEP}] )
+	test? ( dev-cpp/gmock[${MULTILIB_USEDEP}] )"
+# This is provided for backwards compatibility due to (likely incorrect) use in consumers.
+PDEPEND="java? ( dev-java/protobuf-java )
+	python? ( dev-python/protobuf-python[${PYTHON_USEDEP}] )"
+S="${WORKDIR}/${PN}-${MY_PV}"
+PATCHES=( "${FILESDIR}/${PN}-2.5.0-emacs-24.4.patch"
+	"${FILESDIR}/${PN}-2.6.1-protoc-cmdline.patch"
+	"${FILESDIR}/${PN}-3.0.0_beta2-disable-local-gmock.patch" )
+
+src_prepare() {
+	append-cxxflags -DGOOGLE_PROTOBUF_NO_RTTI
```

Comment:
Thanks. I was a bit hesitant since that was one of the few lines I took from the original ebuild, but it seems to be an appropriate change.